cgmanifest.json

```
@@ -7226,6 +7226,16 @@
         }
       }
     },
+    {
+      "component": {
+        "type": "other",
+        "other": {
+          "name": "inih",
+          "version": "47",
+          "downloadUrl": "https://github.com/benhoyt/inih/archive/refs/tags/inih-r57.tar.gz"
```

Should have the real url here
```suggestion
          "downloadUrl": "https://github.com/benhoyt/inih/archive/refs/tags/r57.tar.gz"
```

---

Not sure how i missed this, thanks for catching. Updated